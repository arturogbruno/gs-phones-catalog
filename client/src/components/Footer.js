import React from "react";

const Footer = () => {
    return(
        <div className="Footer">
            <p>Â© Catalog - React Test | Arturo G. Bruno</p>
        </div>
    )
}

export default Footer;