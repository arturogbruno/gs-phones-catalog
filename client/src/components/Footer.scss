.Footer {
    width: 100%;
    height: 50px;
    background-color: #fff;
    box-shadow: 5px 0px 10px 0px rgba(0,0,0,0.5);
    display: flex;
    justify-content: center;
    align-items: center;
    position: absolute; 
    bottom: 0; 
}