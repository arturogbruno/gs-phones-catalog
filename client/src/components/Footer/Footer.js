import React from "react";
import "./Footer.scss";

const Footer = () => {
    return(
        <div className="Footer">
            <p>Â© Catalog - React Test | Arturo G. Bruno</p>
        </div>
    )
}

export default Footer;