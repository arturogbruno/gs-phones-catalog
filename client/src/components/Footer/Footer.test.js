import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

describe("testing Footer component", () => {
    it("should render Footer component", () => {
        expect(shallow(<Footer />)).toMatchSnapshot();
    });
});