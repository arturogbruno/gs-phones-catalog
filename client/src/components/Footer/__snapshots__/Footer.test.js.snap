// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Footer component should render Footer component 1`] = `
<div
  className="Footer"
>
  <p>
    Â© 2020 Catalog - React Test | Arturo G. Bruno
  </p>
</div>
`;
